appgate/__main__.py

```diff
@@ -454,6 +457,11 @@ def main() -> None:
         'Default value: "YYYY-MM-DD_HH-MM-crd.yaml"',
         default=None,
     )
+    dump_crd.add_argument(
+        "--version",
```

Comment:
Don't know if this name is the best one :D Maybe `prefix` or `prefix-version` is cleaner about what it's supposed to be.

Reply:
I will go with `version-suffix` 😄